### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by teckim**
**[VIDEO PRESENTATION](https://www.loom.com/share/84f22142b08f4f568ac69611dd105f33)**
### In this release pull request:

- Created Stock List view.
- Created Stock Details view (company).
- Wrote tests for the components.
- Used `TailwindCSS` for styling.
- Integrated `financialmodelingprep.com` API.
- Deployed the app: [TechStocks on Heroku](https://techstocks-teckim.herokuapp.com/)

> Note: to make this app run, kindly create a `.env.local` file and put the API key there.
> ```
> REACT_APP_FMP_API_KEY=<YOUR_API_KEY_HERE>
> ```
> You can get your API key from [financialmodelingprep](https://site.financialmodelingprep.com/developer/docs/)

### CODE REVIEW

File: src/components/Appbar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { CogIcon, ChevronLeftIcon, MicrophoneIcon } from '@heroicons/react/solid';
+
+const Appbar = () => (
+  <div className="flex items-center h-14 px-2 bg-pink-600">
+    <NavLink className="basis-1/3 flex" to="/">
```
Comment (ShoiraTa) [inline src/components/Appbar.js:7]: - [x] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name. Kudos on following the design guidelines and creating the same layout, however I cannot find the search functionality. 
Kindly add search field that will filter the results by stock name.

File: src/__tests__/appbar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Appbar from '../components/Appbar';
+import renderWithProviders from '../utils/test-utils';
+
```
Comment (ShoiraTa) [inline src/__tests__/appbar.js:3]: - [x]  The application should have: 
-  unit tests for several components of the app and some pure functions in Redux code.
-  integration tests using React Testing Library

Kindly add a couple of unit tests.

File: README.md [hunk 3]
```diff
@@ -26,6 +26,10 @@ To get a local copy up and running follow these simple example steps.
 ### Prerequisites
 NPM should be installed
 
+### ENV variables (`.env.local` file)
+`REACT_APP_FMP_API_KEY`
+> get your API key from [FMP](https://site.financialmodelingprep.com/developer/docs/)
+
 ### Install
 run `npm install` in the project folder.
```
Comment (ShoiraTa) [inline README.md:34]: - [x] Kindly add a link to video presentation with a demo of the project, highlighting all the project features. The video should contain a highlight about a piece of the code you find interesting or proud of making. Add the link to readme, or Pull request.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @teckim ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- Following design guidelines ✔️ 
- App has 2 pages, home and detail ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @teckim, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

179923139-9819ada4-4db2-4c5e-a858-8344d6e4a841


 ## Required Changes ♻️
- [ ]  Kindly record a presentation of your project as requested by the previous reviewer in which you will present the key features of your project as well as a piece of interesting code you worked on while building the project 💯 

## Optional suggestions
- [ ] (OPTIONAL) Even though this is a mobile application, it will be good to create the desktop version to improve the UI/UX of your site.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by teckim** [comment]
> Hi @teckim,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> 179923139-9819ada4-4db2-4c5e-a858-8344d6e4a841 [ 179923139-9819ada4-4db2-4c5e-a858-8344d6e4a841 ](https://user-images.githubusercontent.com/85113567/181255604-55b0d9a1-f1e7-4488-9ac5-5fd4650ebcd4.gif) [ ](https://user-images.githubusercontent.com/85113567/181255604-55b0d9a1-f1e7-4488-9ac5-5fd4650ebcd4.gif)
> 
> ## Required Changes ♻️
> * [ ]   Kindly record a presentation of your project as requested by the previous reviewer in which you will present the key features of your project as well as a piece of interesting code you worked on while building the project 💯
> 
> ## Optional suggestions
> * [ ]  (OPTIONAL) Even though this is a mobile application, it will be good to create the desktop version to improve the UI/UX of your site.
>   _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello @Kingstalux, thank you for your review.
The Video is in the PR, first line.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @teckim     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**